python/fatcat_tools/importers/common.py

```diff
@@ -683,7 +683,7 @@ def run(self) -> Counter:
             xml_path = xml_path.strip()
             if not xml_path or xml_path.startswith("#"):
                 continue
-            with open(xml_path, "r") as xml_file:
+            with open(xml_path) as xml_file:
```

Comment:
I think that the explicit `"r"` is better than relying on the default behavior (which I guess is read-only? but I would need to look it up to confirm).

Reply:
pyupgrade is quite opinionated but these could be manually restored.